terraform/groups/ecs-service/main.tf

```diff
@@ -0,0 +1,72 @@
+terraform {
+  required_version = "~> 1.3.0"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.54.0"
+    }
+    vault = {
+      source  = "hashicorp/vault"
+      version = "~> 3.18.0"
+    }
+  }
+}
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = var.aws_region
+}
+
+module "secrets" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/ecs/secrets?ref=1.0.192"
+
+  name_prefix = "${local.service_name}-${var.environment}"
+  environment = var.environment
+  kms_key_id  = data.aws_kms_key.kms_key.id
+  secrets     = local.parameter_store_secrets
+}
+
+module "ecs-service" {
```

Comment:
You're missing some stuff in here, such as cloudwatch warning etc.. Have a look here: https://github.com/companieshouse/presenter-account-api/blob/main/terraform/groups/ecs-service/main.tf
You don't need the ERIC stuff though, as that's API only. Also you should use the latest version of the ecs-service release